docs/tutorials/open-source/apache-flink/0_setup.qmd

```diff
@@ -0,0 +1,35 @@
+# Getting started
+
+## Overview
+
+[Ibis](https://ibis-project.org/) is a Python library for writing and scaling data transformations. With close to 20 supported backends today, it allows you to write transformations once and reuse the same code for execution across multiple backends (e.g. pandas, Postgres, etc).
```

Comment:
I don't think this (or the next paragraph) are necessary -- would just say that in this tutorial you'll learn how to setup and use Flink w/ Ibis